README.md

```diff
@@ -258,6 +258,44 @@ $ cxxbridge src/main.rs > path/to/mybridge.cc
 
 <br>
 
+## CXX features
```

Comment:
See https://github.com/roligugus/cxx/blob/string_view/README.md#cxx-features for formatted README.

It would probably be nice to have some documentation about features.